submission_runner.py

```diff
@@ -361,12 +372,10 @@ def train_once(
     if (max_global_steps is not None) and (global_step == max_global_steps):
       train_state['training_complete'] = True
 
-    train_step_end_time = time.time()
-    if USE_PYTORCH_DDP:
-      train_step_end_time = sync_ddp_time(train_step_end_time, DEVICE)
+    train_step_end_time = get_time()
 
     train_state['accumulated_submission_time'] += (
-        train_step_end_time - train_step_start_time)
+        train_step_end_time - train_state['last_step_end_time'])
```

Comment:
does this change the semantics of `accumulated_submission_time` to include the evals? IIRC the goal of `accumulated_submission_time` was to just count the time spent in submission functions data selection and update.

Reply:
The train_state['last_step_end_time'] is set to the time after the eval (if there is an eval) line 459. So the accumulated_submission_time should not include the eval time with the code as is now.